strawberryfields/data/fock.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fock backend validation data"""
+from .device import DeviceData
+
+
+class Fock(DeviceData):
```

Comment:
I'd name the DeviceData classes more descriptively, we have so many object names with 'Fock' in them. Maybe "FockData" or "FockCapabilities"?

Reply:
`FockSpecs`?

Reply:
Not bad, maybe also rename the base class to `DeviceSpecs` for consistency?

Reply:
Yep. We could also update the module to `strawberryfields.devicespecs`, for consistency, using `data` feels too general